mock/mock.go

```diff
@@ -252,18 +252,16 @@ func (m *Mock) findClosestCall(method string, arguments ...interface{}) (bool, *
 	return true, closestCall
 }
 
-func callString(method string, arguments Arguments, includeArgumentValues bool) string {
-
-	var argValsString string
-	if includeArgumentValues {
-		var argVals []string
-		for argIndex, arg := range arguments {
-			argVals = append(argVals, fmt.Sprintf("%d: %#v", argIndex, arg))
+func callString(method string, arguments Arguments) string {
+	var argVals []string
+	for i, arg := range arguments {
+		if i > 0 {
+			argVals = append(argVals, ",")
 		}
-		argValsString = fmt.Sprintf("\n\t\t%s", strings.Join(argVals, "\n\t\t"))
+		argVals = append(argVals, fmt.Sprintf("%#v", arg))
```

Comment:
Mmh, I don't really agree. Depending on what the []byte contains, I'd actually preter it to be printed the way `%#v` prints it over printing it as a string. IMO, if you're using a []byte to hold text Go has to manipulate or print, you're doing something wrong. Relevant: https://blog.golang.org/strings. Also, what if the argument is a string nor []byte?

Splitting the arguments makes sense though.